spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,7 @@
+module AuthenticationHelper
+  def authenticate_user(user)
```

Comment:
@ManuSG8 
This will be a convenient helper method, we can use it when we need to authenticate in the system a test user.